server/ipqueue.go

```diff
@@ -189,6 +241,12 @@ func (q *ipQueue[T]) len() int {
 	return l
 }
 
+// Returns the calculated size of the queue (if ipQueue_SizeCalculation has been
+// passed in), otherwise returns zero.
+func (q *ipQueue[T]) size() uint64 {
+	return q.sz.Load()
```

Comment:
This the only reason we made it an atomic? Do we call this function alot?

Reply:
Probably not anymore now that limits are set in the IPQ itself, I can put it back to non-atomic.